downstream/assemblies/platform/assembly-installing-controller-operator.adoc

```diff
@@ -28,6 +28,8 @@ When an instance of {ControllerName} is removed, the associated PVCs are not aut
 == Prerequisites
 
 * You have installed the {PlatformName} catalog in Operator Hub.
+* For Controller, a default 'StorageClass' must be configured on the cluster for the operator to dynamically create needed PVC's. This is not necessary if an external PostgreSQL database is configured.
+* For Hub a 'StorageClass' that supports 'ReadWriteMany' must be available on the cluster to dynamically created the PVC needed for the content, redis and api pods. If it is not the default 'StorageClass' on the cluster, you can specify it when creating your 'AutomationHub' object.
```

Comment:
```suggestion
* For Hub a StorageClass that supports ReadWriteMany must be available on the cluster to dynamically created the PVC needed for the content, redis and api pods. If it is not the default StorageClass on the cluster, you can specify it when creating your 'AutomationHub' object.
```
Same comment as above regarding `ReadWriteMany` and `StorageClass`.

Reply:
So, do those, as they are functions not get any markup at all?

Reply:
@lmalivert I removed them all. I am not used to not calling out functions, that seems odd to me. :-)

Reply:
I took a look at some of the OpenShift docs and neither one of those functions have markup. 
Example for [ReadWriteMany](https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_and_managing_openshift_container_storage_using_google_cloud/managing-persistent-volume-claims_gcp)

Example for [StorageClass](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_and_allocating_storage_resources/storage-classes_rhodf) 

You are the author, so if you disagree, or was advised differently by an SME, disregard my suggestions.